include/wx/maskededit.h

```diff
@@ -0,0 +1,17 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/maskededit.h
+// Purpose:     wxMaskedEdit interface
+// Author:      Manuel Martin
+// Modified by:
+// Created:     2012-07-30
+// RCS-ID:      $Id$
+// Copyright:   (c)
+// Licence:     wxWindows license
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef _WX_MASKED_EDIT_H_BASE
+#define _WX_MASKED_EDIT_H_BASE
+
+#include "wx/generic/maskededit.h"
```

Comment:
This file seems useless, we're very unlikely to have a native implementation of this class, so I'd just move the existing file in `wx/generic` here.